src/client/sky.h

```diff
@@ -134,25 +290,80 @@ class Sky : public scene::ISceneNode
 	bool m_clouds_visible; // Whether clouds are disabled due to player underground
 	bool m_clouds_enabled = true; // Initialised to true, reset only by set_sky API
 	bool m_directional_colored_fog;
-	bool m_bodies_visible = true; // sun, moon, stars
+	bool m_in_clouds = true; // Prevent duplicating bools to remember old values
+
 	video::SColorf m_bgcolor_bright_f = video::SColorf(1.0f, 1.0f, 1.0f, 1.0f);
 	video::SColorf m_skycolor_bright_f = video::SColorf(1.0f, 1.0f, 1.0f, 1.0f);
 	video::SColorf m_cloudcolor_bright_f = video::SColorf(1.0f, 1.0f, 1.0f, 1.0f);
 	video::SColor m_bgcolor;
 	video::SColor m_skycolor;
 	video::SColorf m_cloudcolor_f;
-	v3f m_stars[SKY_STAR_COUNT];
+	
+	// Horizon colors
+	video::SColorf m_bgcolor_day_f = video::SColor(255, 155, 193, 240);
+	video::SColorf m_bgcolor_indoor_f = video::SColor(255, 100, 100, 100);
+	video::SColorf m_bgcolor_dawn_f = video::SColor(255, 186, 193, 240);
+	video::SColorf m_bgcolor_night_f = video::SColor(255, 64, 144, 255);
+	// Sky colors
+	video::SColorf m_skycolor_day_f = video::SColor(255, 140, 186, 250);
+	video::SColorf m_skycolor_dawn_f = video::SColor(255, 180, 186, 250);
+	video::SColorf m_skycolor_night_f = video::SColor(255, 0, 107, 255);
+
+	// pure white: becomes "diffuse light component" for clouds
+	video::SColorf m_cloudcolor_day_f = video::SColorf(1, 1, 1, 1);
+	// dawn-factoring version of pure white (note: R is above 1.0)
+	video::SColorf m_cloudcolor_dawn_f = video::SColorf(
+		255.0f/240.0f,
+		223.0f/240.0f,
+		191.0f/255.0f
+	);
+
+	bool m_default_tint = true;
+	
+	bool m_sun_visible = true;
+	bool m_sunglow_visible = true;
+	std::string m_sun_name = "sun.png";
+	std::string m_sun_tonemap_name = "sun_tonemap.png";
+	std::string m_sunrise_name = "sunrisebg.png";
+	video::SColorf m_pointcolor_sun_f;
+	float m_sun_scale = 1;
+	f32 m_sun_rotation = 90;
+
+	bool m_moon_visible = true;
+	std::string m_moon_name = "moon.png";
+	std::string m_moon_tonemap_name = "moon_tonemap.png";
+	video::SColorf m_pointcolor_moon_f;
+	float m_moon_scale = 1;
+	f32 m_moon_rotation = -90;
+
+	bool m_stars_visible = true;
+	std::vector<v3f> m_stars;
+	float m_star_scale = 1;
+	video::SColor m_starcolor = video::SColor(105, 235, 235, 255);
+	f32 m_star_rotation = 0;
+	u32 m_star_count = 0;
+
 	video::ITexture *m_sun_texture;
+	video::ITexture *m_sunrise_texture;
```

Comment:
unused?

Reply:
Confirmed unused, was originally used for something else, but no longer required.